src/components/events/EventParkingMap.tsx

```
@@ -0,0 +1,219 @@
+            <div className="grid grid-cols-1 sm:grid-cols-3 gap-3 my-auto">
+              {parkingLots.map((lot) => {
+                const status = getParkingOccupancyStatus(lot.currentOccupancy, lot.capacity);
+                const isSelected = lot.id === selectedLotId;
+
+                return (
+                  <div
+                    key={lot.id}
+                    onClick={() => setSelectedLotId(lot.id)}
+                    className={cn(
+                      "p-3 rounded-lg border-2 transition-all cursor-pointer space-y-1.5",
+                      isSelected ? "bg-slate-700 border-amber-400 ring-2 ring-amber-400 scale-105 shadow-lg" : "bg-slate-800/90 border-slate-600 hover:border-slate-400"
```

ESLint

[error] Replace `·?·"bg-slate-700·border-amber-400·ring-2·ring-amber-400·scale-105·shadow-lg"·:·"bg-slate-800/90·border-slate-600·hover:border-slate-400"` with `⏎························?·"bg-slate-700·border-amber-400·ring-2·ring-amber-400·scale-105·shadow-lg"⏎························:·"bg-slate-800/90·border-slate-600·hover:border-slate-400",`

(prettier/prettier)